resqpy/property.py

```diff
@@ -3863,21 +3872,71 @@ def __init__(self, frame=None, property_set_root=None, realization=None):
          for actual well logs, the realization argument will usually be None; for synthetic logs created from an ensemble
          it may be of use
 
-      note:
-         When iterated over, yields instances of :class:`resqpy.property.WellLog` .
+      """
 
-         For example::
+      super().__init__(support=frame, property_set_root=property_set_root, realization=realization)
+
+   def add_log(self, title, data, unit, discrete=False, case_sensitive_units=False, realization=None, write=True):
+      """Add a well log to the collection, and optionally save to HDF / XML
+      
+      Args:
+         title (str): Name of log, typically the mnemonic
+         data (array-like): log data to write. Must have same length as frame MDs
+         unit (str): Unit of measure
+         discrete (bool): by default False, i.e. continuous
+         case_sensitive_units (bool): If True, consider case when parsing units.
+         realization (int): If given, assign data to a realisation.
+
+      Returns:
+         uuids: list of uuids of newly added properties. Only returned if write=True.
```

Comment:
As the method adds exactly one property, might be better to extract the single uuid from the list and return the uuid

Reply:
I think it may return more than one uuid:

```python
collection.add_log(... , write=True)   # Returns 1 uuid

collection.add_log(... , write=False)  # Returns None
collection.add_log(... , write=False)  # Returns None
collection.add_log(... , write=True)   # Returns 3 uuids

```